fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/api/DatatablesApiResource.java

```diff
@@ -225,6 +226,22 @@ public String queryValues(@PathParam("datatable") @Parameter(description = "data
         return this.toApiJsonSerializer.serializePretty(ApiParameterHelper.prettyPrint(uriInfo.getQueryParameters()), result);
     }
 
+    @POST
```

Comment:
This should be a GET request!

Reply:
Hi @adamsaghy  
As this is an advanced filtering we implemented in post method